src/Compilers/VisualBasic/Portable/Symbols/ReferenceManager.vb

```diff
@@ -173,32 +174,56 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Debug.Assert(compilation._lazyAssemblySymbol IsNot Nothing)
             End Sub
 
-            ''' <summary>
-            ''' Creates a <see cref="PEAssemblySymbol"/> from specified metadata. 
-            ''' </summary>
-            ''' <remarks>
-            ''' Used by EnC to create symbols for emit baseline. The PE symbols are used by <see cref="VisualBasicSymbolMatcher"/>.
-            ''' 
-            ''' The assembly references listed in the metadata AssemblyRef table are matched to the resolved references 
-            ''' stored on this <see cref="ReferenceManager"/>. Each AssemblyRef is matched against the assembly identities
-            ''' using an exact equality comparison. No unification Or further resolution is performed.
-            ''' </remarks>
-            Friend Function CreatePEAssemblyForAssemblyMetadata(metadata As AssemblyMetadata, importOptions As MetadataImportOptions) As PEAssemblySymbol
+            '''  <summary>
+            '''  Creates a <see cref="PEAssemblySymbol"/> from specified metadata. 
+            '''  </summary>
+            '''  <remarks>
+            '''  Used by EnC to create symbols for emit baseline. The PE symbols are used by <see cref="VisualBasicSymbolMatcher"/>.
+            '''  
+            '''  The assembly references listed in the metadata AssemblyRef table are matched to the resolved references 
+            '''  stored on this <see cref="ReferenceManager"/>. We assume that the dependencies of the baseline metadata are 
+            '''  the same as the dependencies of the current compilation. This Is Not exactly true when the dependencies use 
+            '''  time-based versioning pattern, e.g. AssemblyVersion("1.0.*"). In that case we assume only the version
+            '''  changed And nothing else.
+            '''  
+            '''  Each AssemblyRef Is matched against the assembly identities using an exact equality comparison modulo version. 
+            '''  AssemblyRef with lower version in metadata Is matched to a PE assembly symbol with the higher version 
```

Comment:
Various instances of `Is` rather than `is`, `Or`, `Not`.

And could the bulk of this comment move to `CommonReferenceManager.GetAssemblyReferenceIdentityBaselineMap` so it's not duplicated?